SolARPipeline_Relocalization/src/SolARRelocalizationPipeline.cpp

```diff
@@ -274,18 +276,42 @@ FrameworkReturnCode SolARRelocalizationPipeline::relocalizeProcessRequest(const
                 processKeyframesId.swap(retKeyframesId);
             else
                 processKeyframesId.insert(processKeyframesId.begin(), retKeyframesId.begin(), retKeyframesId.begin() + NB_PROCESS_KEYFRAMES);
+
+            std::map<uint32_t, std::vector<uint32_t>> mapKeypointCloudPts;
             for (const auto& it : processKeyframesId) {
                 SRef<Keyframe> retKeyframe;
                 m_keyframeCollection->getKeyframe(it, retKeyframe);
                 std::vector<std::pair<uint32_t, SRef<CloudPoint>>> corres2D3D;
                 bool isFound = fnFind2D3DCorrespondences(frame, retKeyframe, corres2D3D);
                 if (isFound) {
```

Comment:
nit: if `isFound` is only used here:
`if ( fnFind2D3DCorrespondences(frame, retKeyframe, corres2D3D) ) {`